OctaneVSPlugin/MainWindowControl.xaml.cs

```diff
@@ -146,10 +146,10 @@ private void ListMenu_Opened(object sender, RoutedEventArgs e)
         private BaseEntity GetSelectedEntity()
         {
             var selectedEntity = SelectedItem?.Entity;
-            if (SelectedItem is CommentViewModel commentViewModel)
+            /*if (SelectedItem is CommentViewModel commentViewModel)
```

Comment:
I would not commit commented code, it's not good practice, git has history so you can always see what was there before